pkg/cloud/amazon/ec2/interface.go

```diff
@@ -0,0 +1,11 @@
+package ec2
+
+import "github.com/jenkins-x/jx/pkg/cluster"
+
+// EC2er is an interface that abstracts the EC2 API
+//go:generate pegomock generate github.com/jenkins-x/jx/pkg/cloud/amazon/ec2 EC2er -o mocks/ec2erMock.go
+type EC2er interface {
```

Comment:
Or just "EC2"?

Reply:
It's convention that interfaces end with `er`.

Reply:
I know. Most often for single-function interfaces. Just sounds a bit odd. That said, better than _option_.